src/app/App.js

```diff
@@ -46,6 +46,10 @@ export default class App {
     return directory.getImplementation(contractName)
   }
 
+  async getProxyImplementation(proxyAddress) {
+    return await this._app.getProxyImplementation(proxyAddress, this.txParams)
```

Comment:
`await` here is not necessary. `async` functions automatically await returned promises.